docs/USAGE.md

```diff
@@ -79,7 +79,7 @@ aws s3 cp --no-sign-request --recursive s3://allencell/aics/morphology_appropria
 Training these models can take days. We've published our trained models so you don't have to. Skip to the [next section](#2-model-inference) if you'd like to just use our models.
 
 1. Create a single cell manifest (e.g. csv, parquet) for each dataset with a column corresponding to final processed paths, and create a split column corresponding to train/test/validation split.
```

Comment:
Should we have info about which columns this manifest must have? Seems related to Yuichiro's question. What do you think?

Reply:
Right now, if you run our preprocessing, then the column name for registered images in `registered_path`, column name for pointclouds is `pointcloud_path`, column name for segs is `seg_path`, and column name for sdfs is `sdf_path`. The name really can be anything, you just have to point to that column name in the config file...Im not sure I want to fix this name to something specific